src/zrc/api/audittrails.py

```diff
@@ -0,0 +1,85 @@
+from django.db import transaction
+
+from ..conf.base import SITE_TITLE
+from ..datamodel.models import AuditTrail
+
+
+class AuditTrailMixin:
+    bron = None
+    main_resource = None
```

Comment:
Let's define an `Audit` class just like `Scope` and `Kanaal` objects. In it, we can store "main_resource" and "component_name" (bron).